contrib/IECoreUSD/test/IECoreUSD/data/customAttribute.usda

```diff
@@ -23,6 +23,7 @@ def Sphere "sphere"
 	)
 	float primvars:arnold:disp_height = 0.5
 	int primvars:arnold:poly_mesh:subdiv_iterations = 3
+	custom string[] test:noAuthoredValue
```

Comment:
Do we know how one generates this in a DCC? What does it mean in the USD spec to have the name with no value?

Reply:
The example I was given was an Atoms crowd cache. I don't think the user had done anything explicitly to get it that way, it just seemed to be something Atoms had done. From reading https://graphics.pixar.com/usd/release/api/class_usd_geom_primvars_a_p_i.html#usdGeom_PrimvarFetchingAPI, I think one interpretation for such an attribute is "discoverable placeholder for something a user might want to author at some point". But I think in this case it's more likely to just be an artifact of the Atoms exporter.

I think you can also achieve a similar state by blocking an attribute : https://graphics.pixar.com/usd/release/api/class_usd_attribute.html#Usd_AttributeBlocking. In that case I think the meaning is intended to be "deleted by stronger layer".